pdf/differential_test/main.py

```diff
@@ -226,7 +196,7 @@ def img_diff(pageno, base, base_tool, img, img_tool, html_out):
 
     diff = None
     if base.shape != img.shape:
-        print(f'Schizophrenic: page {pageno} was size '
+        print(f'Differential: page {pageno} was size '
```

Comment:
It looks like the new `parser-poppler-pdftoppm` has 4 channels instead of 3. Since we're bumping the version number anyhow, we'll want to fix RGB vs RGBA comparisons to *not* report a page size differential, and allow the rest of the differential to proceed by dropping the A channel... may need to blend it with white?

This can be seen on 000009.pdf in the test files, both in HTML and parser version.

Reply:
Sure, now's a good time to fix this.

Reply:
For the files included in the repo, just dropping the alpha channel would be sufficient. I've implemented this as a conversion from alpha to white, though, just in case. I've confirmed this is exactly equivalent for test_files/pdf.